Bylaws.mediawiki

```diff
@@ -80,6 +80,33 @@
 
 # The organization reserves the right to limit membership based on the capacity of the space.
 
+== Equipment ==
+
+Hacklab operates under a general principle of "all members are equal owners of the space". This principle applies to the tools and materials inside of hacklab as well. Unless something has been labeled as property of a specific member, please consider the tool yours. But remember that it is not just yours, it belongs to everybody. Pursuant to this explanation, the following rules can be summarized as:
+
+# Don't use tools in hacklab in a way that will prevent other members from using them. In other words: don't hog or break stuff.
+# Don't hurt yourself.
+
+=== What to do before using equipment ===
+# Don’t use anything without signing the waiver
+# If it can dismember you (or poison you, or otherwise harm you significantly) please don’t use it if you don’t know how
+# If you don’t know how to use it, ask around the membership for help. 
+## Someone brought this equipment in, so someone (if they’re still a member) knows how to use it
+## There may be documentation for it on the public or internal wiki. Check the tool for a QR code. It may link you to a wiki page with instructions.
+
+=== Bringing in New Equipment ===
+# If you want to donate something to the lab, make sure someone (ideally you) is responsible for intaking it. 
+# Whoever takes responsibility for new equipment must:
+## Find a home for it in the lab to keep it permanently. 
+## The home for the new tool should not be on any of the shared workspaces (hot-desks, work-benches in the shop).
+# Ideally, also add documentation for it to the public wiki (knowledge.hacklab.to).
+
+=== Taking equipment from HackLab to borrow and use outside HackLab ===
+# if you want to take something out of the lab, let people know first what you’re taking by messaging a relevant members communications channel. Also give a date for when it will be returned. 
```

Comment:
```suggestion
# If you want to take something out of the lab, message the relevant members communications channel. Let people know first what you're taking and give a date when it will be returned. 
```

Reply:
Yep, looks good.